pages/dkp/kommander/2.1/projects/project-deployments/continuous-delivery/index.md

```diff
@@ -39,11 +37,18 @@ The following table describes the fields required for each authentication method
 
 <!-- Refer https://fluxcd.io/docs/components/source/gitrepositories/#spec-examples for flux examples (not everything in there is supported) -->
 
+If you are using GitOps by using a GitHub repo as your source, you can create your secret with a [personal access token][github-token].
+Then, in the DKP UI, in your project, create a Secret, with a key:value pair of password: <your-token-created-on-github>.
+If you are using a GitHub personal access token, you do not need to have a key:value pair of username: <your-github-username>.
```

Comment:
note that this is true ONLY for Github.

for other scms like gitlab, username is mandatory even if a token (or whatever the scm provider equivalent is) is provided. i havent played with other scms but this is what i read.

we have three options here:
- educate a user github doesn't require username but others might.
- ask the user to use their github username when using a token (tbh, this value is ignored so we are misleading a user here)
- do what you did in this PR - talk about github (and github only), leave out the other scms.

this PR lgtm, just want to provide some context.

Reply:
Good point - I'll push up a note.